curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64faf874364ec308f875f636.md

```diff
@@ -0,0 +1,398 @@
+---
+id: 64faf874364ec308f875f636
+title: Step 38
+challengeType: 0
+dashedName: step-38
+---
+
+# --description--
+
+You need to remove the task from the DOM using `remove()` and from the `taskData` array using `splice()`.
+
+`splice()` is an array method that alters arrays by removing, replacing, or adding elements at a specified index, returning removed elements in the process. It takes up to 3 parameters – the first is the compulsory index to start, the second is the number of items to remove and the third is an optional replacement. Here's an example:
```

Comment:
```suggestion
`splice()` is an array method that modifies arrays by removing, replacing, or adding elements at a specified index, while also returning the removed elements. It can take up to three parameters: the first one is the mandatory index at which to start, the second is the number of items to remove, and the third is an optional replacement element. Here's an example:
```